ui-src/src/event-listeners/index.js

```diff
@@ -14,17 +14,44 @@ import {
   openGoalCreator,
   closeGoalCreator
 } from '../goal-creation/actions'
+import {
+  archiveGoal
+} from '../goals/actions'
+import {
+  archiveEdge
+} from '../edges/actions'
 
 export default function setupEventListeners(store, canvas) {
   document.body.addEventListener('keydown', event => {
     switch (event.code) {
-      case "KeyG":
+      case 'KeyG':
         store.dispatch(setGKeyDown())
         break
-      case "Escape":
+      case 'Escape':
         store.dispatch(closeGoalCreator())
         store.dispatch(unselectAll())
         break
+      case 'Backspace':
+        // archives one goal for now FIXME: should be able to archive many goals
```

Comment:
This next bit is a bit bulky to stay in the event listener file and in this function... maybe a TODO note to refactor that